packages/@pollyjs/adapter-node-http/rollup.config.test.js

```diff
@@ -1,7 +1,17 @@
 import createNodeTestConfig from '../../../scripts/rollup/node.test.config';
 import createJestTestConfig from '../../../scripts/rollup/jest.test.config';
 
-const external = ['http', 'https', 'url', 'stream', 'crypto'];
+const external = [
```

Comment:
Could these be exported and then imported in the rollup config?  This way we're only maintaining a single set of external deps